src/bootstrap-table.js

```diff
@@ -1911,7 +1911,8 @@
             that.trigger(e.type === 'click' ? 'click-row' : 'dbl-click-row', item, $tr, field);
 
             // if click to select - then trigger the checkbox/radio click
-            if (e.type === 'click' && that.options.clickToSelect && column.clickToSelect) {
+            var ignoreTags = [ "A", "BUTTON" ];
+            if (e.type === 'click' && that.options.clickToSelect && column.clickToSelect && !ignoreTags.includes(e.target.tagName)) {
```

Comment:
Lower version browser does not support `includes`, use `$.inArray` method instead.

Reply:
Sure thing.